swift/ListDemo/ListDemo/ListDemoApp.swift

```diff
@@ -0,0 +1,21 @@
+//
+//  ListDemoApp.swift
+//  ListDemo
+//
+//  Created by Chr1s on 2022/2/21.
+//
+
+import SwiftUI
+
+@main
+struct ListDemoApp: App {
+    var body: some Scene {
+        WindowGroup {
+            ContentView()
+                .onAppear {
+                    // 去除多余的打印信息
+                    UserDefaults.standard.setValue(false, forKey: "_UIConstraintBasedLayoutLogUnsatisfiable")
```

Comment:
:exclamation:**Cycode**: Secret of type: 'Generic Password' was found.
Severity: High

Would you like to exclude this secret from your status checks?
Tell us what to do with one of the following hashtags:

| Tag | Short Description |
| --- | --- |
| **#cycode_secret_ignore_here** | Applies to this request only |
| **#cycode_secret_ignore_everywhere** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_revoked** | Applies to this secret value for all repos in your organization |
| **#cycode_secret_false_positive** | Applies to this secret value for all repos in your organization |

 <!-- c09bbc1c-11dd-4b80-b8ba-29cd6891852b -->